node/src/command.rs

```diff
@@ -144,8 +146,8 @@ macro_rules! construct_async_run {
 		let runner = $cli.create_runner($cmd)?;
 		runner.async_run(|$config| {
 			let $components = new_partial::<
-				RuntimeApi,
-				Executor,
+				picasso_runtime::RuntimeApi,
+				PicassoExecutor,
```

Comment:
we'll need to abstract away the client over any generic runtime, otherwise we can't execute the right sub commands. this is going to be more involved than i thought